legacy/cal/include/common.php

```diff
@@ -19,8 +19,8 @@
 # you'll have to do some heavy editing to the year-specific version the
 # festival calendar (e.g., "viewpp2010.php").
 define("PPNAME", "Pedalpalooza");	    # Name of the festival
-define("PPSTART", "2019-06-01");	    # Start date, in MySQL format
-define("PPEND", "2019-06-30");		    # End date, in MySQL format
+define("PPSTART", "2023-06-01");	    # Start date, in MySQL format
+define("PPEND", "2023-08-31");		    # End date, in MySQL format
 define("PPSTART_MONTHDAY", "June 1");	    # Start date, in "Month Day" format
 define("PPDAYS", 30);			    # Duration in days
```

Comment:
I don't think we're currently using`PPDAYS` or `PPDATES`, but might as update those too. Could update the `PPSMALL` and `PPLARGE` image URLs also; I don't think we use them via this context so it probably doesn't really matter, but we do update the images year over year.

Reply:
So, there are still a bunch of code references to this, that get whacked into some pretty questionable mysql statements.  :)  I think the only risk here is some other code starts working again, but anything that depends on MONTHDAY or PPDATES just won't work on a multi-month scenario.

Reply:
If nothing's been noticeably broken since 2019 regarding this, we're probably aok.